thirdparty/cmdconfig/commands/fnevalcmd_test.go

```diff
@@ -229,7 +157,7 @@ apiVersion: v1
 		{
 			name: "custom kind with storage mounts",
 			args: []string{
-				"run", "dir", "--mount", "type=bind,src=/mount/path,dst=/local/",
+				"eval", "dir", "--mount", "type=bind,src=/mount/path,dst=/local/",
```

Comment:
Where's the behavior change for validating that either a mount or exec are provided?

Reply:
https://github.com/GoogleContainerTools/kpt/blob/f6fc94d7a3591b9ec8d05c1bec2577be1edd2643/thirdparty/kyaml/runfn/runfn.go#L474

mount is only used with container runtime.

Reply:
I don't see the behavior change for validating in the fn changes doc.

Reply:
What @frankfarzan probably meant is that `mount`, `network` and `as-user` flags only makes sense with `--image` arg not with `--exec-path`.

Reply:
Added checks for these flags.